lib/class/update.class.php

```diff
@@ -3362,126 +3355,41 @@ public static function update_370030()
         return $retval;
     }
 
-    /**
-     * update_370031
+    /*
+     * update_380000
+     *
+     * Adds ability to store movie MPAA and TV content ratings
+     * Chamged  tvshow 'summary' to "overview" and Increased size to 1024
+     * Changed tvshow_episode summary size to 1024.
      *
-     * Add an option to customize login art, favicon and text footer.
      */
-    public static function update_370031()
+    public static function update_380000()
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [The method update_380000 is not named in camelCase.](https://www.codacy.com/app/paul/ampache/file/616134498/issues/source?bid=7631&fileBranchId=2290754#l3366)